### PR TITLE
[Doc] Add software freedom to objectives

### PR DESCRIPTION
**Pull request by McSinyx**
Resolve GH-136.

### CODE REVIEW

File: docs/source/intro.rst [hunk 1]
```diff
@@ -92,10 +92,15 @@ Constructing an Applicable Web Application
 """"""""""""""""""""""""""""""""""""""""""
 
 The system should be implemented as a web application for
-portability |~| [mikkonen2008]_.  It should be easy to be adopted by users on
+portability. |~| [mikkonen2008]_  It should be easy to be adopted by users on
```
Comment (Huy-Ngo) [inline docs/source/intro.rst:95]: Is there a convention to cite after or before the sentence ends?

**Review by Huy-Ngo — APPROVED** [review]
:+1:

File: docs/source/intro.rst [hunk 2]
```diff
@@ -150,8 +155,12 @@ The report contains following chapters:
 .. [mikkonen2008] Tommi Mikkonen and Antero Taivalsaari.
    "Web Applications---Spaghetti Code for the 21st Century".
    *2008 Sixth International Conference on Software Engineering Research,
-   Management and Applications*, p.|~| 319--328, Prague, 2008.
+   Management and Applications*, p. |~| 319--328, Prague, 2008.
    :doi:`10.1109/SERA.2008.16`.
+.. [libredu] Richard Stallman.
+   "Why Schools Should Exclusively Use Free Software".
+   *Free Software and Education*.  GNU Project.  Retrieved 2021-02-01.
+   https://www.gnu.org/education/edu-schools.html
```
Comment (Huy-Ngo) [inline docs/source/intro.rst:163]: No idea if we can cite websites but since GNU is famous I think it's acceptable.

**Comment by McSinyx** [comment]
Tone-wise, the article sounds more like a free form essay
than a formal research but it's the most canon source I found.

File: docs/source/intro.rst (hunk 1 above)
Comment (McSinyx) [reply docs/source/intro.rst:95]: [No](https://www.quora.com/When-citing-a-source-at-the-end-of-a-sentence-should-you-put-the-period-before-the-set-of-parentheses-or-after?share=1), it was late, lemme revert.

**Pull request merged**